app.js

```diff
@@ -6,20 +6,43 @@
 // a number of minutes as an argument and returns
 // the number of seconds
 
-function convertToSeconds() {}
+function convertToSeconds(minutes) {
```

Comment:
this can be refactored too! let me know if you want to work through that or if you have questions.

Reply:
yes, please!